### ISSUE TITLE
Fix resolution of diagrams in figure 4 and figure 5

### ISSUE THREAD

**Issue by jauxillos** [DimmestP/chimera_project_manuscript#166]
There is a problem in the resolution of the diagrams (the original files are fine but when its put into the paper, it looks blurry/pixelated
 
Suggestion to address this - 
- Either we upload a jpg of the entire diagram to get better resolution of the figuree
or
- figure out how to improve the resolution

**Comment by jauxillos** [comment]
Same probleem in figure S2, S3 and S4 (secondary structures)

**Comment by ewallace** [comment]
I did not fix this in the pull request #178 because I didn't know how and I ran out of time.

I have 3 suggestions:
* adjust the call to `ggdraw` to change the resolution when loading svg
* adjust the call to `ggsave` to change the resolution when saving, possibly through `fig_dpi` paramter in `shared_figure_formatting.R`
* load the file in a different format, possibly pdf?

To this stack overflow issue has some ideas.
https://stackoverflow.com/questions/50026862/the-draw-image-function-from-cowplot-results-in-blurred-pdfs
